asynciterator.ts

```diff
@@ -1084,7 +1084,7 @@ export class TransformIterator<S, D = S> extends BufferedIterator<D> {
     if (!source || source.done || (item = source.read()) === null)
       done();
     else if (!this._optional)
-      this._transform(item, next);
+      this._transform(item, next, this._push.bind(this));
```

Comment:
probably want a stored `item => this._push(item)` for performance (I can update)
To basically, in the constructor/member variable declaration, `this._pushBound = item => this._push(item)`.

Reply:
+1 to storing. I wasn't aware that `.bind` is less performant than a lambda.

Reply:
👍 will keep